### PR TITLE
Add Disconnect(Node) Menu And Node Header Tooltip

### PR DESCRIPTION
**Pull request by MYTNB**
# Expanded the disconnect Menu Items of <NodePort>
# add Node header tooltip

### CODE REVIEW

**Comment by Siccity** [comment]
Please move GraphEditor.GetNodeTooltip to NodeEditor.GetHeaderTooltip

**Comment by MYTNB** [comment]
> Please move GraphEditor.GetNodeTooltip to NodeEditor.GetHeaderTooltip

Okay, I have submitted this modification

File: Scripts/Editor/NodeEditor.cs [hunk 1]
```diff
@@ -104,6 +104,11 @@ public class NodeEditor : XNodeEditor.Internal.NodeEditorBase<NodeEditor, NodeEd
             return NodeEditorResources.styles.nodeHighlight;
         }
 
+        /// <summary> Override to display custom node header tooltips </summary>
+        public virtual string GetHeaderTooltip(XNode.Node node) {
+            return null;
+        }
+
```
Comment (Siccity) [inline Scripts/Editor/NodeEditor.cs:111]: Now that it's in NodeEditor, there is no reason to pass the node as we already have it as `target`.
Please remove the node target parameter

Comment (MYTNB) [reply Scripts/Editor/NodeEditor.cs]: You’re right, I didn’t think much about it when move the code

**Comment by Siccity** [comment]
Thanks! 👍

**Comment by ChuKuang** [comment]
> Thanks! 👍
form disconnect menu  disconnect node has bug.
image

**Comment by MYTNB** [comment]
> > Thanks! 👍
> > form disconnect menu  disconnect node has bug.
> > image

Looks like a legacy problem @Siccity 
I think it’s safer to use this method。 @ChuKuang 
image

**Pull request merged**